native~/Editor/src/SelectIonTokenWindowImpl.cpp

```diff
@@ -274,6 +303,7 @@ void updateDefaultToken(
 
     server.defaultIonAccessToken(System::String(response.value->token));
     server.defaultIonAccessTokenId(System::String(response.value->id));
+    UnityEditor::EditorUtility::SetDirty(server);
 
     // TODO: source control
```

Comment:
I think I left this comment in a long while ago and forgot to take it out :sweat_smile: will do that now